src/cluster/src/delete.rs

```diff
@@ -248,13 +248,32 @@ impl ClusterUninstaller {
         Ok(())
     }
 
+    /// Check whether fluvio-sys is already installed
+    fn is_sys_installed(&self) -> Result<bool, UninstallError> {
+        let sys_chart_installed = self
+            .helm_client
+            .get_installed_chart_by_name("fluvio-sys", Some(&self.config.namespace))
+            .map_err(UninstallError::HelmError)?
+            .iter()
+            .nth(0)
+            .is_some();
+
+        Ok(sys_chart_installed)
+    }
+
     /// in order to remove partitions, finalizers need to be cleared
+    #[instrument(skip(self))]
     async fn remove_partitions(&self, namespace: &str) -> Result<(), UninstallError> {
```

Comment:
this won't work for object objects as well.  Instead,  change `remove_objects` to ignore errors